tests/mocks/tendermint_tm_db_DB.go

```diff
@@ -5,9 +5,10 @@
 package mocks
 
 import (
+	reflect "reflect"
```

Comment:
Why is this alias? :D Is autogenerated?

Reply:
yea its autogen, everything within tests/mock is autogen

Reply:
ohh I shouldn't have touched that